### PR TITLE
Remove env prefix

### PR DESCRIPTION
**Pull request by halink0803**

### CODE REVIEW

**Comment by halink0803** [comment]
[Remove environment variable prefix](https://trello.com/c/97iqZsAR/286-remove-environment-variable-prefix)

File: docker-compose.yml [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
     volumes:
-      - ./data/postgres:/var/lib/postgresql/data
+      - './data/postgres:/var/lib/postgresql/data'
     environment:
       POSTGRES_DB: reserve_stats
       POSTGRES_USER: reserve_stats
       POSTGRES_PASSWORD: reserve_stats
     ports:
-      - "127.0.0.1:5432:5432"
+      - '127.0.0.1:5432:5432'
```
Comment (favadi) [inline docker-compose.yml]: I guess this is OK, but why the changes?

Comment (favadi) [reply docker-compose.yml]: merging anyway.
